http/src/test/scala/org/http4s/blaze/http/http2/ProtocolFrame.scala

```diff
@@ -8,7 +8,7 @@ sealed abstract class ProtocolFrame private extends Product with Serializable
 object ProtocolFrame {
   case class GoAway(lastHandleStream: Int, cause: Http2Exception) extends ProtocolFrame
 
-  case class Ping(data: Option[Array[Byte]]) extends ProtocolFrame
+  case class Ping(isAck: Boolean, data: Array[Byte]) extends ProtocolFrame
```

Comment:
Cleaner, are Ping and Ack really the same Frame? That seems odd, but the new repr is cleaner.

Reply:
They are exact mirrors with the exception of the ACK flag, this previously it was actually not a true representation. The 8 bytes are typically a stamp of some sort and may have meaning to the client, for example, to disentangle multiple pings, although the server _should_ send acks in the same order as pings are received.